### PR TITLE
Add addVarying to ShaderBuilder

### PR DESCRIPTION
**Pull request by ptrgags**
This PR addresses the additional feedback @lilleyse left us in https://github.com/CesiumGS/cesium/pull/9648#pullrequestreview-698630455

@sanjeetsuhag could your review?

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thanks for the pull request @ptrgags!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/CesiumGS/cesium/blob/shader-builder-revisions/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by lilleyse** [comment]
Looks good from a quick look. @sanjeetsuhag all you for the full review / merge.

File: Source/Renderer/ShaderBuilder.js [hunk 1]
```diff
@@ -224,6 +226,23 @@ ShaderBuilder.prototype.addAttribute = function (type, identifier) {
   return location;
 };
 
+/**
+ * Add a varying declaration to both the vertex and fragment shaders.
+ *
+ * @param {String} type The GLSL type of the varying
+ * @param {String} identifier An identifier for the varying. Identifiers must begin with <code>v_</code> to be consistent with Cesium's style guide.
+ */
```
Comment (sanjeetsuhag) [inline Source/Renderer/ShaderBuilder.js:234]: Add an example in the documentation here, like you've done with `addUniform` and `addAttribute`

File: Source/Renderer/ShaderBuilder.js [hunk 2]
```diff
@@ -63,11 +63,13 @@ export default function ShaderBuilder() {
     defineLines: [],
     uniformLines: [],
     shaderLines: [],
+    varyingLines: [],
```
Comment (sanjeetsuhag) [inline Source/Renderer/ShaderBuilder.js:66]: Use `addVarying` in the example in the documentation.

**Review by sanjeetsuhag — COMMENTED** [review]
@ptrgags This change looks good, just a couple of documentation changes needed to match the other functions.

**Comment by ptrgags** [comment]
@sanjeetsuhag updated!

**Review by sanjeetsuhag — APPROVED** [review]
Everything looks good. Thanks @ptrgags!

**Pull request merged**